Hotails/settings.py

```diff
@@ -15,7 +15,6 @@
 # Build paths inside the project like this: BASE_DIR / 'subdir'.
 BASE_DIR = Path(__file__).resolve().parent.parent
 
-
```

Comment:
This change is not related to this PR, you can move to another commit or do it on a follow up PR.

Reply:
this line was generated by Django for us.
You can see it already in the settings.py file that was merged